scripts/src/tools/gitutils.py

```diff
@@ -44,10 +45,26 @@ def set_git_username_email(repo, username, email):
 def github_api_post(endpoint, bot_token, headers={}, json={}):
     r = requests.post(f'{GITHUB_BASE_URL}/{endpoint}',
                       headers=headers, json=json)
+
+    try:
+        response_json = r.json()
+
+        if "message" in response_json:
+            print(f'[ERROR] getting secret: {response_json["message"]}')
+            sys.exit(1)
+    except json.decoder.JSONDecodeError:
+        pass
+
+
     return r
 
 def github_api_get(endpoint, bot_token, headers={}):
     r = requests.get(f'{GITHUB_BASE_URL}/{endpoint}', headers=headers)
+    response_json = r.json()
+    if "message" in response_json:
+        print(f'[ERROR] getting secret: {response_json["message"]}')
```

Comment:
We can improve the error message saying `[ERROR] During GET Request :`

Reply:
good spot, fixed it.